tasks/section5.yml

```diff
@@ -131,15 +153,32 @@
       - rule_5.1.7
 
 - name: "SCORED | 5.1.7 | PATCH | Ensure permissions on /etc/cron.d are configured"
-  command: /bin/true
+  file:
+      dest: /etc/cron.d
+      state: directory
+      owner: root
+      group: root
+      mode: 0600
   tags:
       - level1
       - level2
       - patch
       - rule_5.1.7
 
 - name: "SCORED | 5.1.8 | AUDIT | Ensure at/cron is restricted to authorized users"
-  command: /bin/true
+  shell: stat -L /etc/at.deny > /dev/null
+  register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*